### PR TITLE
[MIG] 16.0: stock_picking_procure_method

### PR DESCRIPTION
**Pull request by micheledic**
MIG of stock_picking_procure_method from 14.0

### CODE REVIEW

**Comment by rousseldenis** [comment]
/ocabot migration stock_picking_procure_method

@micheledic Thanks for this. Could you follow migration guide ? https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0#technical-method-to-migrate-a-module-from-150-to-160-branch

Thanks

**Comment by micheledic** [comment]
> /ocabot migration stock_picking_procure_method
> 
> @micheledic Thanks for this. Could you follow migration guide ? https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0#technical-method-to-migrate-a-module-from-150-to-160-branch
> 
> Thanks

Done! can you check ?

**Comment by carolinafernandez-tecnativa** [comment]
@micheledic Could you please squash administrative commits? Thanks!

**Comment by pedrobaeza** [comment]
Or at least rebase for getting a fresh runboat.

**Comment by micheledic** [comment]
@carolinafernandez-tecnativa @pedrobaeza rebased and squashed commits!

**Review by carolinafernandez-tecnativa — APPROVED** [review]
TT45185

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

File: stock_picking_procure_method/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
     "name": "Stock Picking Procure Method",
     "summary": "Allows to force the procurement method from the picking",
-    "version": "14.0.1.0.0",
+    "version": "16.0.0.0.0",
```
Comment (rousseldenis) [inline stock_picking_procure_method/__manifest__.py:6]: @pedrobaeza This one is not correct

Comment (pedrobaeza) [reply stock_picking_procure_method/__manifest__.py:6]: Ouch, I didn't notice. I will do direct commit, as it will trigger the same the wheel creation.

**Pull request merged**